runtests.sh

```diff
@@ -4,4 +4,4 @@
 # If you are getting an INVOCATION ERROR for this script then there is 
 # a good chance you are running on Windows.
 # You can and should use WSL for running tox on Windows when it calls bash scripts.
-REQUESTS_CA_BUNDLE=`python -m pytest_httpbin.certs` pytest $*
+REQUESTS_CA_BUNDLE=`python -m pytest_httpbin.certs` exec pytest "$@"
```

Comment:
```suggestion
REQUESTS_CA_BUNDLE=`python3 -m pytest_httpbin.certs` exec pytest "$@"
```

Reply:
@jspricke good idea but we have more places calling python rather than python3 in the codebase. Let's fix all of them at once in a dedicated pull request after merge maybe?

Reply:
fine with me.